app/models/project.rb

```diff
@@ -516,7 +516,13 @@ def download_registry_users
   def self.find_with_includes!(platform, name, includes)
     # this clunkiness is because .includes() doesn't allow zero args and we want
     # to allow that.
-    query = self.visible.platform(platform).where(name: name)
+    project_find_names = begin
+       "PackageManager::#{platform.capitalize}".constantize.project_find_names(name)
+    rescue => exception
```

Comment:
Same question as above

Reply:
Here the rescue is meant to raise a `RecordNotFound` error in place of an error of not being able to find the `PackageManager` class.